gptme/message.py

```diff
@@ -211,6 +211,15 @@ def format_msgs(
         if highlight:
             color = ROLE_COLOR[msg.role]
             userprefix = f"[bold {color}]{userprefix}[/bold {color}]"
+        if msg.role == "system":
+            first_line = msg.content.split("\n", 1)[0].lower()
+            isSuccess = first_line.startswith(("saved", "executed", "ran", "appended", "patch successfully"))
+            isError = first_line.startswith(("error", "failed","missing arguments", "unknown command", "no patch"))
```

Comment:
I see tmux gives "missing arguments", the message should probably be changed to "Error: Missing arguments" so the detection can be simplified.
Same for "unknown command" and "no patch".

Reply:
Done